fairscale/nn/data_parallel/sharded_ddp.py

```diff
@@ -9,26 +9,19 @@
 Adopted from LegacyDistributedDataParallel module from fairseq.
 """
 
-from collections import OrderedDict
 from contextlib import contextmanager
 import copy
-from typing import TYPE_CHECKING, Any, Dict, Generator, List, Optional, cast
+from typing import Any, Dict, Generator, List, Optional, Type, cast
 
 import torch
-from torch import nn
+from torch import Tensor, nn
 import torch.distributed as dist
+from torch.nn import Parameter
 
-if TYPE_CHECKING:
-    from fairscale.optim import OSS
-    from torch import Tensor
-    from torch.nn import Parameter
-else:
-    OSS = Any
-    Tensor = Any
-    Parameter = Any
+from fairscale.optim import OSS
```

Comment:
is OSS going to be renamed to ShardOptimizer at some point?

Reply:
yes I would love to, at least to match the RFC entirely. On FB side this means that we would need to touch classy and lightning code, but I was not sure about fairseq so I erred on the side of caution and did not touch it. @msbaines , thoughts ?